src/radical/pilot/agent/executing/popen.py

```diff
@@ -201,6 +201,8 @@ def _handle_unit(self, cu):
             # Start a new subprocess to launch the unit
             self.spawn(launcher=launcher, cu=cu)
 
+            time.sleep(1)
```

Comment:
Why is this delay needed? This also assumes that all CUs will be more than a second.

Reply:
Thanks, removed.